rg_shipit/classes/ShipitOrigin.php

```diff
@@ -0,0 +1,82 @@
+<?php
+  class ShipitOrigin {
+    public $street = '';
+    public $number = '';
+    public $complement = '';
+    public $commune_id = '';
+    public $full_name = '';
+    public $email = '';
+    public $phone = '';
+    public $store = false;
+    public $origin_id = null;
+    public $name = null;
+
+    public function __construct($street = '', $number = '', $complement = '', $commune_id = '', $full_name = '', $email = '', $phone = '', $store = false, $origin_id = null, $name = null) {
```

Comment:
Why in this constructor do you assign variables as a parameter if it is a constructor? they receive and assign the parameters

Reply:
I'm not sure this works that way... If i'm correct, that are just default values that takes place on assignment when that field is not beign passed by... just default values for not getting errors like null pointer exceptions or such

Reply:
i think it was a plugin that I used to build that class, I'll fix it (but if it works)